src/components/daydream-controls.js

```diff
@@ -83,7 +81,14 @@ module.exports.Component = registerComponent('daydream-controls', {
   },
 
   checkIfControllerPresent: function () {
-    var isPresent = this.isControllerPresent(this.el.sceneEl, GAMEPAD_ID_PREFIX, {hand: this.data.hand});
+    var handedness = this.data.hand === '' ? ['left', 'right'] : [this.data.hand];
```

Comment:
Nope.  I realize this, I haven't had time to address this yet.  As per our discussion, I think that `isControllerPresent` needs to still return true with empty hand, which it doesn't right now.